projects/ccf-rui/src/app/modules/info/info-dialog/info-dialog.module.ts

```diff
@@ -1,12 +1,16 @@
-import { NgModule } from '@angular/core';
+import { NgModule, SecurityContext } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { InfoDialogComponent } from './info-dialog.component';
 import { MatDialogModule } from '@angular/material/dialog';
 import { MatIconModule } from '@angular/material/icon';
+import { MatExpansionModule } from '@angular/material/expansion';
+import { MarkdownModule } from 'ngx-markdown';
 
 @NgModule({
   declarations: [InfoDialogComponent],
-  imports: [CommonModule, MatDialogModule, MatIconModule],
+  imports: [CommonModule, MatDialogModule, MatIconModule, MatExpansionModule, MarkdownModule.forRoot({
+    sanitize: SecurityContext.NONE
```

Comment:
Curious, why is this necessary?

Reply:
So Angular automatically sanitizes the DOM when you're inputting elements in it (to prevent XSS) so without this, I was getting errors that didn't allow the parsing to happen. But turns out removing that now doesn't cause any errors. I'll remove this.